packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -0,0 +1,223 @@
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/tooltip/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/tooltip"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-tooltip--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for you to use in production. Components such as Tooltip are likely to not see any API
+    changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol. Feel free to share
+    your experience using this component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>. We are
+    also looking for a contributor to symbolize this component in Sketch. Interested? Reach out!
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Tooltip
+
+The Tooltip component is a popup that displays text related to an element when the element is focused via
+the keyboard or on mouse hover. Tooltip follows the
+[WAI-ARIA Tooltip Pattern](https://www.w3.org/TR/wai-aria-practices/#tooltip).
+
+## Examples
+
+### Basic Tooltip
+
+<LivePreview
+  scope={{
+    Tooltip,
+    Button,
+  }}
+>
+  {`<Tooltip baseId="tooltip" text="Black Lives Matter. We stand with the Black community.">
+  <Button variant="primary">✊ Action</Button>
+</Tooltip>`}
+</LivePreview>
+
+### Controlling placement
+
+The placement of the tooltip is configurable via the `placement` prop. The available placement options are
+available in the [props description here](#tooltip). By default the tooltip is positioned automatically so
+you don't have to worry about it going off the page, or being covered by another element.
+
+<LivePreview
+  scope={{
+    Tooltip,
+    Button,
+  }}
+>
+  {`<Tooltip baseId="tooltip-right" text="Welcome to Paste!" placement="right-start">
+  <Button variant="primary">Open Tooltip</Button>
+</Tooltip>`}
+</LivePreview>
+
+### Focusable element
+
+A tooltip can be placed on any **focusable** element, such as a button or a React component with a `ref`. Since our
+Box primitive passes a `ref`, it can be used to wrap a non-focusable element such as an icon. If you do use an icon,
+make sure you set the `decorative` prop to false, and give the icon a title. This ensures the icon is accessible
+to screen readers.
+
+<Callout variant="warning">
+  <CalloutText>
+    🚨<strong>CAUTION!</strong>🚨 Make sure not to abuse this method. Tooltips should only be placed on elements that
```

Comment:
So, I'm a little confused by what you're trying to say here as the example does what we say not to do.

This could be fixed by wrapping the icon in the example with an anchor, but it must be a natively focusable html element.

Reply:
@SiTaggart I've the example up. Removed this callout and simplified the text above. I also added a callout below about icon titles.